docs/man/skstat.8.adoc

```diff
@@ -283,6 +280,9 @@ If this router is not a neighbor, this field identifies the next-hop neighbor us
 link::
 The ID of the link to the neighbor router.
 
+ver::
+The inter-router protocol version used by the router.
```

Comment:
, for example AMQP 1.0?

didn't see this in output:

```
kubectl exec --stdin --tty deploy/skupper-router -- skstat -n

2023-01-11 11:18:15.425186 UTC
east-skupper-router-6c8866fc67-nc448

Routers in the Network
Last Topology Change: Wednesday Jan 11 10:49:07 2023 GMT
  router-id                             next-hop  link
  ======================================================
  east-skupper-router-6c8866fc67-nc448  (self)    -
  west-skupper-router-79664f4686-qbmnx  -         0

```

Reply:
It appears if you use `skstat -nv`.

Reply:
The version does not refer to `AMQP 1.0`.  It refers to the skupper-router-specific inter-router protocol used to discover topology and share information about address destinations.